rpms/s3cmd/cortx-s3-s3cmd.spec

```diff
@@ -70,20 +70,17 @@ s3cmd is a CLI based s3-client.
 
 %install
 rm -rf $RPM_BUILD_ROOT
-S3CMD_PACKAGING=Yes python3.6 setup.py install --prefix=%{_prefix} --root=$RPM_BUILD_ROOT
+S3CMD_PACKAGING=Yes python setup.py install --prefix=%{_prefix} --root=$RPM_BUILD_ROOT
```

Comment:
Did you check is it fine to use  python2 here??

Reply:
yes.Had discussion with Nilesh yesterday and to remove inconsistency between rpm version we decided to use python 2 version